docs/INSTALL.md

```diff
@@ -478,7 +478,7 @@ zmqpubrawtx=tcp://127.0.0.1:28333
 
 Once all of the above is complete, and you've confirmed `bitcoind` is fully
 updated with the latest blocks on testnet, run the command below to launch
-`lnd` with `bitcoind` as your backend (as with `bitcoind`, you can create an
+`lnd` with `bitcoind` as your backend (as with `bitcoind`, you can create a
```

Comment:
If I understand English correctly I believe this should be an `an`.

Reply:
unaddressed